src/elements/TabbedContainer/TabbedContainer.js

```diff
@@ -0,0 +1,99 @@
+import './TabbedContainer.less';
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Div from '../Div/Div';
+import List from '../List/List';
+
+class TabbedContainer extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			// set default showing tab (0 = first tab, 1 = second tab, etc...)
+			activeIndex: 0
+		};
+
+		this.changeTabs = this.changeTabs.bind(this);
+		this.renderNavItems = this.renderNavItems.bind(this);
+		this.renderOneNavItem = this.renderOneNavItem.bind(this);
+	}
+
+	changeTabs(event, key) {
+		event.preventDefault();
+
+		this.setState({
+			activeIndex: key
+		});
+	}
+
+	renderNavItems(key) {
+		const tab = this.props.children[key] ? this.props.children[key] : null,
+			keyNum = Number(key);
+
+		const navItemClasses = classNames(
+			this.state.activeIndex === keyNum ? 'active' : ''
+		);
+
+		return (
+			<li key={key} className={navItemClasses}>
+				<a href="#" onClick={ (e) => this.changeTabs(e, keyNum) }>{ tab.props.title }</a>
+			</li>
+		);
+	}
+
+	renderOneNavItem() {
+		const oneTab = React.Children.only(this.props.children);
+
+		return (
+			<li className="active">
+				<a href="#">{ oneTab.props.title }</a>
+			</li>
+		);
+	}
+
+	render() {
+		let index = 0,
+			renderNav;
```

Comment:
`renderNav` is used for storing the rendered nav elements, so best practices would be to use a noun to describe this, rather than a verb (which makes it seem like it will be a function).

Also, variables are best declared right before assigned them. Since `const` and `let` are block scope you _do_ need to declare them outside of `if/else` statements, but they should be done right before that block appears.